ref/general/java-se.adoc

```diff
@@ -10,13 +10,13 @@
 :page-type: general
 = Java SE support
 
-Open Liberty requires a Java SE runtime in order to run. A Java SE runtime (also known as a Java Runtime Environment, or a Java Developer Kit), provides the tools to develop, run and monitor your Java application. Open Liberty can run on any Java SE 8 compatible runtime, or the Java SE 11 runtimes listed below. Due to behaviour and API changes between Java SE 8 and Java SE 11 it is possible that an application that runs on Open Liberty/Java SE 8 may not run on Open Liberty/Java SE 11, see the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for more details.
+Open Liberty requires a Java SE runtime in order to run. A Java SE runtime (also known as a Java Runtime Environment, or a Java Developer Kit), provides the tools to develop, run and monitor your Java application. Open Liberty can run on any Java SE 8, 11, or 12 compatible runtime listed below. Due to behaviour and API changes between Java SE 8 and Java SE 11 it is possible that an application that runs on Open Liberty/Java SE 8 may not run on Open Liberty/Java SE 11, see the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for more details.
```

Comment:
Change
"Open Liberty requires a Java SE runtime in order to run. A Java SE runtime (also known as a Java Runtime Environment, or a Java Developer Kit), provides the tools to develop, run and monitor your Java application. Open Liberty can run on any Java SE 8, 11, or 12 compatible runtime listed below. Due to behaviour and API changes between Java SE 8 and Java SE 11 it is possible that an application that runs on Open Liberty/Java SE 8 may not run on Open Liberty/Java SE 11, see the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for more details."
to
"Open Liberty requires a Java SE runtime. A Java SE runtime (also known as a Java Runtime Environment or a Java Developer Kit), provides the tools to develop, run, and monitor your Java application. Open Liberty can run on any Java SE 8, 11, or 12 compatible runtime. Because of behavior and API differences between Java SE 8 and Java SE 11, an application that runs on Open Liberty with Java SE 8 might not run on Open Liberty with Java SE 11. See the https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide] for details."

Reply:
I'm going to keep the "compatible runtime listed below" phrasing, because we don't support any compatible Java SE 8, 11, or 12 runtime, just the specific versions listed below (e.g. Oracle 8u25+)

Reply:
I understand what you're saying, and that's fine. "Below" doesn't pass accessibility tests, so we just try to eliminate it wherever possible.